apps/graphql/src/apps/itinerary/types/outputs/Itinerary.js

```diff
@@ -14,30 +14,32 @@ export type Itinerary = {|
   +id: string,
 |};
 
-export default new GraphQLObjectType({
-  name: 'Itinerary',
-  fields: {
-    destination: { type: GraphQLLocation },
-    endTime: { type: GraphQLDateType },
-    id: GlobalID(({ id }) => id),
-    origin: { type: GraphQLLocation },
-    price: { type: GraphQLPrice },
-    sectors: {
-      type: new GraphQLList(GraphQLSector),
-      resolve: ({ sectors }: ItinerariesType): Sector[] => {
-        if (sectors == null) {
-          return [];
-        }
-        return sectors.map((sector, index) => ({
-          ...sector,
-          stopoverDuration: getStopoverDuration(sector, sectors[index - 1]),
-        }));
-      },
+const itineraryResponseTypes = {
```

Comment:
What was your reason to put this out from the `GraphQLObjectType`?

Reply:
Just for my information

Reply:
@FilipMessa Hey! because on an earlier version I created two different objects, one for each query, and I reused the fields, but now I just use one so we can put it back in.

Reply:
OK I understand, we can have it outside, but I am just not sure about that name. 
From my perspective, it is fields not typed.